### PR TITLE
feature/add-journal-cover-to-articles-without-doi-BZ-5406

### PR DESCRIPTION
**Pull request by tiandavis**
## Summary - [BZ-5406](https://thirdiron.atlassian.net/browse/BZ-5406)

Adds journal cover image to article search results with a journal issn but no article doi.

### Description

TOOD:
- Review Regression Tests (DONE)

## Deploy Precautions

- None

### CODE REVIEW

**Comment by karlbecker** [comment]
image

image


Sorry @tiandavis , I couldn't resist, it popped in my head right away 😂

File: tests/unit/models/summon.js [hunk 1]
```diff
@@ -279,6 +290,17 @@ describe("Summon Model >", function() {
...
+
+    it("should build a journal endpoint for an article search result with a journal issn but no article doi", function() {
+      var scope = {
+        document: {
+          content_type: "Journal Article",
+          issns: ["0028-4793"]
+        }
+      };
+
+      expect(summon.getEndpoint(scope)).toContain("search?issns=00284793");
```
Comment (karlbecker) [inline tests/unit/models/summon.js]: So glad we've got all these tests to help clarify the requirements and prevent regressions. Great job updating and adding more to these, Tian! 👏 👏

Comment (tiandavis) [reply tests/unit/models/summon.js]: `npm test` 236 tests and growing...

File: src/summon/browzine-summon-adapter.js [hunk 2]
```diff
@@ -84,12 +84,17 @@ browzine.summon = (function() {
...
+      endpoint = api + "/search?issns=" + issn;
+    }
+
+    if(isArticle(scope) && getDoi(scope)) {
       var doi = getDoi(scope);
       endpoint = api + "/articles/doi/" + doi + "?include=journal";
     }
 
-    if(isJournal(scope)) {
+    if(isArticle(scope) && !getDoi(scope) && getIssn(scope)) {
```
[the remark this answers is not in the document]
Comment (tiandavis) [reply src/summon/browzine-summon-adapter.js]: Good question Karl! I don't have a single clue. 😅

[the remark this answers is not in the document]
Comment (karlbecker) [reply src/summon/browzine-summon-adapter.js]: Any thoughts from you @johnseguin ?

**Comment by johnseguin** [comment]
@karlbecker Good thinking - and the answer is _maybe_.  Conference Proceedings might be labeled differently, but we don't cover those right now anyways, so I don't think we're missing much there. :)  I'd rather keep the scope more narrow as opposed to inadvertently lighting up a bunch of ISSN's in records for things we don't really understand yet and have people asking why this random cover is on this item, etc, etc. :)  Thanks!

**Comment by karlbecker** [comment]
Thanks for the clarification @johnseguin !

**Pull request merged**